packages/governance/src/types.js

```diff
@@ -351,6 +351,13 @@
  * @property {ParamValue} proposedValue
  */
 
+/**
+ * @typedef {Object} ApiInvocationIssue
+ * @property {string} methodName
+ * @property {Instance} contract
+ * @property {any} methodParams
```

Comment:
Is this accurate?
```suggestion
 * @property {Record<string, Callable>`} methodParams
```
(`Callable` is defined in `@agoric/eventual-send`)

Reply:
I think I really meant Array<any>. What did you intend to convey with `Callable`?

 (I presume the single quote is just a typo, and not some syntax I haven't learned yet.)  

[after discussion]

I think we ended up with something that should mean an Array whose type will come from the contract. I put in `{[unknown]}`, hoping that that's how one writes a TBD tuple.